src/engine/Util/Util.ts

```diff
@@ -104,7 +104,7 @@ module ex.Util {
    }
 
    export function clamp(val, min, max) {
```

Comment:
Could you add the `number` types to each parameter to make sure people don't accidentally pass in other types?